uptime_service_validation/coordinator/helper.py

```diff
@@ -367,3 +367,10 @@ def getExistingNodes(conn, logger):
     finally:
         cursor.close()
     return nodes
+
+
+def sendSlackMessage(url, message, logger):
+    payload='{"text": "%s" }' % message
+    response = requests.post(url, data=payload)
+    logger(response)
```

Comment:
Not 100% sure, but I think `logger.info(reponse)` is correct. Unless you tested it?

Reply:
Nice catch. Thanks.